src/traces/contour/make_color_map.js

```diff
@@ -21,6 +21,11 @@ module.exports = function makeColorMap(trace) {
         nc = Math.floor((end - start) / cs) + 1,
         extra = contours.coloring === 'lines' ? 0 : 1;
 
+    if(!isFinite(cs)) {
```

Comment:
Hoo boy. Okay, so. I've tried to piggyback to the full extent possible on existing contour behavior. Part of that was to use, for example, `start = 5`, `end = Infinity`, `size = Infinity` for the inequality `z > 5`. That causes it to automatically extract the correct contour levels without additional logic. So in that sense, the information is 'correct'. The only caveat is that it required a couple additional checks in places like this.